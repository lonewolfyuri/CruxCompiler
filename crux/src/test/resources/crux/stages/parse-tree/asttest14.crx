// parameter and local of same name

func foo(a:int) : void {
  var a : bool;
  ::println();
}
