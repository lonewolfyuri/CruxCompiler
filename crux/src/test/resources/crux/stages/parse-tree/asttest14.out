program
  declarationList
    declaration
      functionDefinition
        parameterList
          parameter
            type
        type
        statementBlock
          statementList
            statement
              variableDeclaration
                type
            statement
              callStatement
                callExpression
                  expressionList
